scripts/new-solr-updater.py

```diff
@@ -54,9 +56,14 @@ def get_default_offset():
 
 
 class InfobaseLog:
-    def __init__(self, hostname):
+    def __init__(self, hostname, exclude=None):
+        """
+        :param str hostname:
+        :param str exclude: if specified, excludes records that include the string
```

Comment:
Just food for thought...

Once we drop legacy Python, we can use Python [type hints](https://docs.python.org/3/library/typing.html) of the form:
`def __init__(self, hostname: str, exclude: str=''):`
And [mypy](http://mypy-lang.org/) with be able to use these type hints to lint the code the usage of incorrect types.

If you folks believe that we would be interested to go that way in the future then this code would to use an empty string instead of None.  (Mypy would complain that None is not a str.)

Reply:
👍 For type hints and mypy! That's one thing I'm really excited about in Py3 :) For this specific case, I'd rather this be of type `Optional[str]`; using `''` for the `None` feels like a bit of a code smell, but I haven't looked into it too much.